front/src/applications/stdcmV2/components/StdcmConsist.tsx

```diff
@@ -118,6 +144,31 @@ const StdcmConsist = ({ disabled = false }: StdcmConfigCardProps) => {
           onSelectSuggestion={onSelectSuggestion}
         />
       </div>
+      <div className="towed-rolling-stock">
+        <ComboBox
+          id="towedRollingStock"
+          label={t('consist.towedRollingStock')}
+          value={towedRsFilters.text.toUpperCase()}
+          onClick={() => {
+            if (selectedTowedRollingStock?.id !== undefined) {
+              searchRollingStockById(selectedTowedRollingStock.id);
+            }
+          }}
+          onChange={(e) => {
+            searchTowedRollingStock(e.target.value);
+            if (e.target.value.trim().length === 0) {
```

Comment:
```suggestion
            if (!e.target.value.trim()) {
```
Small enhancement but not mandatory :)